### ISSUE TITLE
add accessible help menu for diff editor

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#185028]
We can likely just tweak the current one for the editor

This would allow us to replace the hint about `shift+f7` with a help menu hint and we could teach users what to expect wrt audio cues etc.

Blocked on the diff editor refactor

**Comment by meganrogge** [comment]
@hediet it should be pretty easy to understand, lmk if you have questions. here's Aaron's adoption in the notebook as an example

https://github.com/microsoft/vscode/pull/186229

**Comment by hediet** [comment]
Verification steps:
* Press Alt+F1 when in diff editor. Verify that this opens a help menu explaining how to open the accessible diff viewer.
